### ISSUE TITLE
about init_fn for multi gpu training

### ISSUE THREAD

**Issue by liuheng92** [open-mmlab/mmaction2#1145]
Before raising a question, you may need to check the following listed items.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmaction2.readthedocs.io/en/latest/faq.html) but cannot get the expected help.

hi, there. I realised when I used train.py script training in multi-gpu mode, worker_init_fn function may produce same seed in each processing. So dataloader may produce same data in each processing each epoch.

**Comment by liuheng92** [comment]
https://github.com/pytorch/pytorch/pull/56488#issuecomment-825128350

**Comment by dreamerlin** [comment]
According to https://github.com/open-mmlab/mmaction2/blob/master/mmaction/datasets/builder.py#L110-L112, you can set seed when you launch a job

**Comment by liuheng92** [comment]
When I use default seed or input fixed seed, that will be a problem in small dataset.  So I should input a random seed, like torch.initial_seed(), right?

**Comment by liuheng92** [comment]
Did't cause much gap when I run my own experiments. So I'll close this.
